src/document.rs

```diff
@@ -6,10 +6,14 @@ use std::collections::HashMap;
 use std::fmt::Write;
 use std::path::{Path, PathBuf};
 
+pub(crate) const CRLF_NEWLINE: &str = "\r\n";
```

Comment:
As this is binary, i.e. leaf node in the dependency graph, `pub(crate)` and `pub` are equivalent.